communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "ballista.png",
+      "title": "Ballista Distributed Compute",
+      "quote": "Join our Discord channel to chat with the core contributors.",
```

Comment:
The `"quote"` is intended to be about your experience with Discord, would you mind writing a short blurb about your experience using Discord?